OpenNefia.Content/Skills/SkillPrototype.cs

```diff
@@ -8,12 +8,30 @@
 
 namespace OpenNefia.Content.Skills
 {
+    public enum SkillType
+    {
+        Undefined,
+
+        StatSpecial,
+        Stat,
+        WeaponProficiency,
+        Skill,
+        SkillMagic,
+        SkillAction,
+        SkillEffect
+    }
     [Prototype("Skill")]
     public class SkillPrototype : IPrototype, IHspIds<int>
     {
         [DataField("id", required: true)]
         public string ID { get; } = default!;
 
+        [DataField]
+        public PrototypeId<SkillPrototype>? RelatedSkill { get; }
+
+        [DataField]
```

Comment:
Should be `[DataField(required: true)]`, and the default should be `Skill`.

Reply:
done in 76f6762